Sources/_StringProcessing/Regex/Core.swift

```diff
@@ -76,6 +76,11 @@ public protocol RegexComponent<RegexOutput> {
 /// instances using a clear and flexible declarative syntax. Using this
 /// style, you can combine, capture, and transform regexes, `RegexBuilder`
 /// types, and custom parsers.
+///
+/// > Note:
+/// > Prior to Swift 6,
+/// > you might need to write `#/myregex/#` instead of `/myregex/`
+/// > when you make a regular expression using a literal.
```

Comment:
```suggestion
/// > when you make a regular expression using a literal.
/// > For more information,
/// > see [Regular Expression Literals][regex-literal] in  *[The Swift Programming Language][tspl]*.
///
/// [regex-literal]: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/lexicalstructure/#Regular-Expression-Literals
/// [tspl]: https://docs.swift.org/swift-book/
```

Reply:
@natecook1000 Worth adding a cross reference?  Copying the phrasing from other xrefs to TSPL.